pages/api/tags/get-all-tags.js

```diff
@@ -0,0 +1,46 @@
+import * as Utilities from "~/node_common/utilities";
+import * as Data from "~/node_common/data";
+
+export default async (req, res) => {
+  const id = Utilities.getIdFromCookie(req);
+  if (!id) {
+    return res.status(500).send({ decorator: "SERVER_GET_SLATES_TAGS", error: true });
+  }
+
+  const user = await Data.getUserById({
+    id,
+  });
+
+  if (!user) {
+    return res.status(404).send({
+      decorator: "SERVER_GET_SLATE_USER_NOT_FOUND",
+      error: true,
+    });
+  }
+
+  if (user.error) {
+    return res.status(500).send({
+      decorator: "SERVER_GET_SLATE_USER_NOT_FOUND",
+      error: true,
+    });
+  }
+
+  const response = await Data.getSlatesByUserId({ userId: id });
+  if (!response) {
+    return res.status(404).send({ decorator: "SERVER_GET_SLATES_NOT_FOUND", error: true });
+  }
+
+  if (response.error) {
+    return res.status(500).send({ decorator: "SERVER_GET_SLATES_NOT_FOUND", error: true });
+  }
+
+  const filteredTags = response
```

Comment:
Also, post-database migration I have an optimization that will make this a lot faster, but for now we can leave it like this. I also notice that this doesn't get file level tags. Is that intentional? (getting file level tags will also be easier post migration)

Reply:
I decided to remove this endpoint altogether because it is not needed now. 

Also, I made a mistake by not getting file level tags. This has been fixed now.